Sources/armory/logicnode/GetMeshCulledNode.hx

```diff
@@ -0,0 +1,18 @@
+package armory.logicnode;
+
+import iron.object.Object;
+
+class GetMeshCulledNode extends LogicNode {
+
+	public function new(tree: LogicTree) {
+		super(tree);
+	}
+
+	override function get(from: Int): Dynamic {
+		var object: Object = inputs[0].get();
+
+		if (object == null) false;
```

Comment:
This line doesn't make a lot of sense, did you mean `return` instead of `false`? The same goes for the other nodes.

Reply:
It is because i made this node based in `Get Object Visible`, is not the same method?

Reply:
I think it's a mistake there, too. This statement just doesn't have any effect whatsoever and is probably even optimized away by the compiler.

Reply:
I also have fixed Get Visible. Thanks to notice it.

Reply:
When i change the line `if (object == null) return false;` to `if (object == null) return;`, i got `Void should be Dynamic` error. What is wrong?

Reply:
The same happens to `Get Visible`. I think there is a reason why it is with `false` instead of `return`

Reply:
Oh, my bad. Then it is `return false` instead of just `false`. `return` would just return nothing (`Void`) but the function signature requires that something is returned.

Reply:
Alright, i will fix it. Thanks :+1: